Enterprise-CMCS/macpro-mdct-hcbs
# LTSS-4 Measure Overview Page

### Description
This PR adds the content for the LTSS-4 measure.
As of now, thanks to the all the great work done in setting up [LTSS-1](https://github.com/Enterprise-CMCS/macpro-mdct-hcbs/pull/66), I only had to update the qm.ts and cmit.ts files.

#### For this requirement (same as LTSS-3): 
> I propose, holding off on the edit button going to a blank page at least until the routing by page id PR is merged in, so there's consistency with how we're routing.
 - Clicking the edit button should take you to that measure's delivery method page.

Notes from LTSS-1
1. Children textboxes of the radio button currently does not save any data. Address in a separate PR.
2. The quality measure table [Edit] buttons do not go anywhere yet. Address in a separate PR.

### Related ticket(s)
CMDCT-4145

---
### How to test

1. Sign into HCBS, any state user
2. Create a QM report
3. On the sidebar menu, select [Optional Measure Results]. 
   - The 5th row down will be LTSS-4, CMIT # 962.
4. Click [Edit] and you be taken to the LTSS-4 measure overview page.
5. Check that the questions matches what is written in the jira ticket: [CMDCT-4145](https://jiraent.cms.gov/browse/CMDCT-4145) - same as LTSS-1 & LTSS-3
6. You should see a page that looks like this
    ![LTSS-4](https://github.com/user-attachments/assets/b023a719-d67a-4f35-a276-9d08b0efdc2c)



### Important updates

---
### Author checklist


---
